app/components/static-page.hbs

```diff
@@ -6,7 +6,7 @@
     sizes="50vw"
     src="/assets/images/loket-header-1600.jpg"
     srcset="/assets/images/loket-header-320.jpg 320w, /assets/images/loket-header-1024.jpg 1024w, /assets/images/loket-header-1600.jpg 1600w"
-    alt="" role="presentation"
+    alt="een laptop met daarop het vlaanderen icoon"
```

Comment:
We removed that alt text by choice, actually. This seems to be a decorative image and I don't think a person using a screenreader needs or wants to know that information?

Reply:
Ok, removed the alt text. It's a bit strange to use role="presentation" (removes the image completely from the accessibility API and an empty alt text is enough) here but it seems that the linter requires it.

Reply:
Yes, I agree. I discussed this in the ember discord a11y channel and the person responsible for the a11y linting rules ensured me that it was working as intended. After a linking to specific parts of the spec she backtracked and said the rule needed to be updated but I don't think that happened yet.